Source/WebCore/html/HTMLMediaElement.h

```diff
@@ -1342,6 +1344,7 @@ class HTMLMediaElement
 
     std::optional<DynamicRangeMode> m_overrideDynamicRangeMode;
     PlatformDynamicRangeLimit m_platformDynamicRangeLimit;
+    bool m_shouldSuppressHDR { false };
```

Comment:
This PR doesn't appear to support the case when a video element is created while HDR is suppressed. We also need some way to test this.

Reply:
That case is now supported, as well as creating new pages.
I'm still working on adding tests...

Reply:
Now with tests.